tests/func/test_api.py

```diff
@@ -8,7 +8,7 @@
 from dvc.main import main
 from dvc.path_info import URLInfo
 from dvc.utils.fs import remove
-from tests.remotes import GCP, HDFS, OSS, S3, SSH, Azure, Local
+from tests.remotes import GCP, HDFS, OSS, S3, SSH, Azure, GDrive, Local
 
 remote_params = [S3, GCP, Azure, OSS, SSH, HDFS]
```

Comment:
We should be able to simply add Gdrive to this list. Sorry for not mentioning it explicitly earlier.

Reply:
yes wasn't sure if the other tests would pass - wanted to debug individually but changed now.